lib/OpenQA/Parser/Format/LTP.pm

```diff
@@ -30,7 +30,13 @@ sub parse {
         $t_name =~ s/:/_/g;
 
         $result->{result} = 'ok';
-        $result->{result} = 'fail' if $result->{test}->{result} !~ /pass/i || $result->{status} !~ /pass/i;
+
+        if ($result->{test}->{result} =~ /(fail|brok)/i || $result->{status} =~ /(fail|brok)/i) {
```

Comment:
```suggestion
        if ($result->{test}->{result} =~ /(fail|broken)/i || $result->{status} =~ /(fail|broken)/i) {
```

We are not being charged per character :)

Reply:
No we're not, but LTP apparently was, because it has `TBROK` in result output:

in both C API:

https://github.com/linux-test-project/ltp/blob/7fe464cd8443a99adabd188e098f34e539f41511/include/tst_res_flags.h#L12-L13

and shell API:

https://github.com/linux-test-project/ltp/blob/7fe464cd8443a99adabd188e098f34e539f41511/testcases/lib/tst_test.sh#L104-L105

matching `broken` would miss these `TBROK` results.

Reply:
NOTE: the legacy LTP API prints `CONF`, `BROK`, new API prints `TCONF`, `TBROK`, that's why missing leading `T`.

Reply:
Believe me, the original code was correct, we don't want to introduce a regression :).

Reply:
So, if it's a known bug, make sure there's a comment for this, and the other thingie with conf, and I'm good

Reply:
@acerv Are you still planning to add a comment here?

Reply:
@kalikiana I will do it now

Reply:
@kalikiana done